shared/fs/footer/upload-container.js

```diff
@@ -1,16 +1,42 @@
 // @flow
-import {connect, type TypedState} from '../../util/container'
+import * as FsGen from '../../actions/fs-gen'
+import * as Types from '../../constants/types/fs'
+import {compose, connect, setDisplayName, type Dispatch, type TypedState} from '../../util/container'
 import Upload, {type UploadProps} from './upload'
 import {formatDurationFromNowTo} from '../../util/timestamp'
 
 const mapStateToProps = (state: TypedState) => ({
   _uploads: state.fs.uploads,
 })
 
-const mergeProps = ({_uploads}, dispatchProps, ownProps) =>
+// NOTE flip this to show a button to debug the upload banner animations.
+const enableDebugUploadBanner = true
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  debugToggleShow: !enableDebugUploadBanner
+    ? undefined
+    : (() => {
```

Comment:
lets not put this in here inline. just make a closure up top. and likely wrap this in a if (__DEV__)